cicd/builds/basic-build-operations.adoc

```diff
@@ -17,6 +17,7 @@ include::modules/builds-basic-cancel-multiple.adoc[leveloffset=+2]
 include::modules/builds-basic-cancel-all.adoc[leveloffset=+2]
 include::modules/builds-basic-cancel-all-state.adoc[leveloffset=+2]
 include::modules/builds-basic-delete-buildconfig.adoc[leveloffset=+1]
+include::modules/builds-basic-edit-buildconfig.adoc[leveloffset=+1]
```

Comment:
IMHO I would recommend the order edit, delete (instead of delete, edit) here

Reply:
Done.